controllers/helmchartproxy/helmchartproxy_controller_phases.go

```diff
@@ -70,22 +70,29 @@ func (r *HelmChartProxyReconciler) reconcileForCluster(ctx context.Context, helm
 		// TODO: Should we set a condition here?
 		return errors.Wrapf(err, "failed to get HelmReleaseProxy for cluster %s", cluster.Name)
 	}
-	// log.V(2).Info("Found existing HelmReleaseProxy", "cluster", cluster.Name, "release", existingHelmReleaseProxy.Name)
 
-	if existingHelmReleaseProxy != nil && shouldReinstallHelmRelease(ctx, existingHelmReleaseProxy, helmChartProxy) {
-		log.V(2).Info("Reinstalling Helm release by deleting and creating HelmReleaseProxy", "helmReleaseProxy", existingHelmReleaseProxy.Name)
-		if err := r.deleteHelmReleaseProxy(ctx, existingHelmReleaseProxy); err != nil {
-			conditions.MarkFalse(helmChartProxy, addonsv1alpha1.HelmReleaseProxySpecsUpToDateCondition, addonsv1alpha1.HelmReleaseProxyDeletionFailedReason, clusterv1.ConditionSeverityError, "%s", err.Error())
+	if helmChartProxy.Spec.ReconcileStrategy == string(addonsv1alpha1.ReconcileStrategyInstallOnce) {
+		if internal.HasHelmReleaseBeenSuccessfullyInstalled(existingHelmReleaseProxy) {
+			log.Info("HelmReleaseProxy has been installed on InstallOnce mode, nothing to do", "helmReleaseProxy", existingHelmReleaseProxy.Name, "cluster", cluster.Name)
```

Comment:
This line too--just confirming that we want `log.Info` without an explicit log level.